uiowa/uiowa
# Use photoswipe instead of colorbox

Resolves https://github.com/uiowa/uiowa/issues/4669
Resolves https://github.com/uiowa/uiowa/issues/3695
Resolves https://github.com/uiowa/uiowa/issues/4744 

## ToDo

- [x] fix link contrast in captions (links are dark blue on dark gray)
- [x] test v2/v3 masonry - https://github.com/uiowa/uiowa/issues/4744
- [x] remove colorbox dependency and from config - removed dependencies (GA tracked colorbox) BUT removing from composer creates chaos. Will look further into it
- [x] create and use patch file for the photoswipe grouping instead of using the MR patch file
- [x] require a particular dev commit? - see recent drupal/cshs as an example
- [x] Revert security check disablement in blt config
- [x] overnight or cache clear if we are doing all this in one PR?
- punt image gallery woes for another day!
TM a11y check/approval - put in DEV with a test gallery for TM to look at - emailed TM, waiting for feedback/response #4777 

## Testing Instructions

1. composer install
2. blt ds --site=sandbox.uiowa.edu
3. blt frontend
4. drush @sandbox.local cr
5. drush @sandbox.local uli
6. add an image gallery block - click on images and make sure captions are readable
7. now test using the masonry style
7. now sync down a v2 site  (blt ds --site=brand.uiowa.edu)
8. drush @brand.local uli
9. add an image gallery paragraph - click on images and make sure captions are readable
10. now test with masonry style
